schema-engine/connectors/schema-connector/src/warnings.rs

```diff
@@ -416,6 +418,12 @@ impl fmt::Display for Warnings {
             f
         )?;
 
+        render_warnings(
+            "The following models are marked as capped collections, which are not yet fully supported. Read more: https://pris.ly/d/mongodb-capped-collections",
```

Comment:
CLI Warning

Reply:
(Equivalent to what we choose above I guess)

Reply:
Does this one need to change? I thought this one was fine as is

Reply:
I find "marked as" a bit weird, as we use "is" in the other one.
But yeah, does not matter to much.

Reply:
```suggestion
        render_warnings(
            "The following models are capped collections, which are not yet fully supported. Read more here: https://pris.ly/d/mongodb-capped-collections",
```

Reply:
I just realised if we updated this to read as `Read more here: ...` that would be inconsistent with the other warnings but I honestly prefer including the here so am inclined to update the other messages to have it too, thoughts?

Reply:
It is currently as follows:
```
The following models are capped collections, which are not yet fully supported. Read more: https://pris.ly/d/mongodb-capped-collections
```